tasks/condition/condition.js

```diff
@@ -4,35 +4,67 @@
  * Task - rewrite IF..ELSE to ternary operator
  */
 {
-  const a = 3,
-        b = 2;
+const a = 3,
+     b = 2;
 
-  if (a + b < 4) {
+ if (a + b < 4) {
     console.log('wrong');
-  } else {
-    console.log('too much');
+ } else {
+   console.log('too much');
   }
 }
 
+const a = 3,
+        b = 2;
+
+ const condvalue = (a + b < 4) ? 'wrong' : 'too much';
+    console.log(condvalue);
+
 /** TODO
  * The system receives two params x and y.
  * Implement the logic when z equal to sum of x and y in case both of them less than 10 or multiplication in case of greater or equal than 10
  * Task 1 - if else
  * Task 2 - ternary operator
  */
-{
-    const x = 3;
+{const x = 3;
     const y = 7;
-}
+    if (x<10, y<10) {
```

Comment:
this is a bad practice, please use logic operators (&& or ||),
if ( x < 10 && y < 10) {...}